src/task-6.js

```diff
@@ -1,7 +1,21 @@
 
 export default function getStats(data) {
-    // Change me!
-
+    if (data.length !== 0) {
+        let mn = data[0]; // у разі використання Math.min не потрібно
```

Comment:
Just a proposal here. We can use the same variables names as we must return in an object. In such case, we can use property shorthand in object literal. More info https://eslint.org/docs/rules/object-shorthand.